lib/rspec/core/configuration.rb

```diff
@@ -2135,6 +2135,8 @@ def load_file_handling_errors(method, file)
         suggestions = DidYouMean.new(relative_file).call
         reporter.notify_non_example_exception(ex, "An error occurred while loading #{relative_file}.#{suggestions}")
         RSpec.world.wants_to_quit = true
+      rescue SyntaxError => e
+        raise e
```

Comment:
Should we add `RSpec.world.wants_to_quit = true` here to skip loading other examples?